src/core/server/execution_context/execution_context_container.test.ts

```diff
@@ -16,11 +16,11 @@ import {
 
 describe('KibanaExecutionContext', () => {
   describe('constructor', () => {
-    it('allows context to define parent explicitly', () => {
+    it.skip('allows context to define parent explicitly', () => {
```

Comment:
Maybe we should rewrite the test or remove it altogether?

Reply:
fixed